style.css

```diff
@@ -12,6 +12,37 @@ html {
     text-align: center; 
 }
 
+#header-bg {
+    background-color: var(--background-color-header);
+    height: 80px;
+}
+
+#header {
+    margin: 0em 5em;
+    min-width: auto;
+    max-height: auto;
+}
+
+/* NEED HELP HOW TO ALGIN LIST
+    Mr-DanielDang */
+#header ul {
+    list-style: none;
+}
+
+#header li {
```

Comment:
could you explain why you need to set li elment to display inline?

Reply:
To display the list horizontally